src/lib.rs

```diff
@@ -187,6 +192,36 @@ impl Device {
         }
     }
 
+    pub fn usb_purge_tx_buffer(&mut self) -> Result<()> {
+        let result = unsafe { ffi::ftdi_usb_purge_tx_buffer(self.context) };
+        match result {
+            0 => Ok(()),
+            -1 /* read */ | -2 /* write */ => Err(Error::RequestFailed),
```

Comment:
Documentation [disagrees](https://www.intra2net.com/en/developer/libftdi/documentation/ftdi_8c.html#a1eab1dfd17068c46775a86496fee8bec):
```
0 | all fine
-1 | read buffer purge failed
-2 | USB device unavailable
```
where "device unavailable" is likely the "uninitialized context" case.

Reply:
Done